rental.rb

```diff
@@ -0,0 +1,18 @@
+# Class rental
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+    @book = ''
+    @person = ''
+  end
+
+  def add_book(book)
+    book.add_rental(self)
+  end
+
+  def add_person(person)
+    person.add_rental(self)
+  end
+end
```

Comment:
- [ ] For the `Rental` class, I would suggest you push an instance of it into the rentals array of the `Book` and the `Person` rentals array. Remember that the `Rental` class is sort of the intermediary between the `Book` and `Person` classes because it belongs to those classes. So you need to set this many-to-many relationship in its constructor by adding itself to these classes. 

_**Hint:**_

```
def initialize(date, book, person)
    .
    .

    @book = book
    book.rentals << self

    @person = person
    person.rentals << self
  end
```
```